src/components/cc-zone/cc-zone.js

```diff
@@ -87,8 +94,8 @@ export class CcZone extends LitElement {
             <span class="title ${classMap({ skeleton })}">${title}</span>
             <span class="subtitle ${classMap({ skeleton })}">${subtitle}</span>
           </div>
-          ${infraProviderSlug != null ? html`
-            <cc-img class="infra-logo" src=${ifDefined(getInfraProviderLogoUrl(infraProviderSlug))} text=${ifDefined(infraProviderSlug)}></cc-img>
+          ${infra != null ? html`
+            <cc-img class="infra-logo" src=${ifDefined(getInfraProviderLogoUrl(infra))} text=${ifDefined(infra)}></cc-img>
```

Comment:
**🎯 suggestion:** I think we can safely remove the `ifDefined` directive here (both usage), as `infra` will always be defined.

Reply:
`infra` can still be `null` if you look closely at the `CcZone._getInfraProviderSlug` function

Reply:
My bad, I should've been more clear: it will always be defined considering the predicate line 97 `infra != null ? ...` outputting the HTML.

Reply:
You're absolutely right, I've just fixed that